backend/ReactTypeOfSideEffect.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+// Copied from React repo.
+
+module.exports = {
+  NoEffect: 0, //           0b00000000
+  Bailout: 1, //            0b00000001
+  Placement: 2, //          0b00000010
+  Update: 4, //             0b00000100
+  PlacementAndUpdate: 6, // 0b00000110
+  Deletion: 8, //           0b00001000
+  ContentReset: 16, //      0b00010000
+  Callback: 32, //          0b00100000
+  Err: 64, //               0b01000000
+  Ref: 128, //              0b10000000
+};
```

Comment:
> I'm not worried about Bailout number changing because I left a comment in React itself about it, and React repo logic also depends on it being 1 specifically.

Agreed. 😄